<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title></title>
  <meta name="description" content="">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="index.html">
<link rel="next" href="changes-of-gene-frequency.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Notebook for QG</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Preface</a></li>
<li class="chapter" data-level="2" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html"><i class="fa fa-check"></i><b>2</b> Genetic Constitution of A Population</a><ul>
<li class="chapter" data-level="2.1" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#frequencies-of-genesalleles-and-genotypes"><i class="fa fa-check"></i><b>2.1</b> Frequencies of genes(alleles) and genotypes</a><ul>
<li class="chapter" data-level="2.1.1" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#mendelian-variation-in-natural-populations"><i class="fa fa-check"></i><b>2.1.1</b> Mendelian variation in natural populations</a></li>
<li class="chapter" data-level="2.1.2" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#causes-of-change"><i class="fa fa-check"></i><b>2.1.2</b> Causes of change</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#hardy-weinberg-equilibrium"><i class="fa fa-check"></i><b>2.2</b> Hardy-Weinberg equilibrium</a><ul>
<li class="chapter" data-level="2.2.1" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#the-hardy-weinberg-law"><i class="fa fa-check"></i><b>2.2.1</b> The Hardy-Weinberg law</a></li>
<li class="chapter" data-level="2.2.2" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#applications-of-the-hardy-weinberg-law"><i class="fa fa-check"></i><b>2.2.2</b> Applications of the Hardy-Weinberg law</a></li>
<li class="chapter" data-level="2.2.3" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#mating-frequencies-and-another-proof-of-the-hardy-weinberg-law"><i class="fa fa-check"></i><b>2.2.3</b> Mating frequencies and another proof of the Hardy-Weinberg law</a></li>
<li class="chapter" data-level="2.2.4" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#multiple-alleles"><i class="fa fa-check"></i><b>2.2.4</b> Multiple alleles</a></li>
<li class="chapter" data-level="2.2.5" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#sex-linked-genes"><i class="fa fa-check"></i><b>2.2.5</b> Sex-linked genes</a></li>
<li class="chapter" data-level="2.2.6" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#more-than-one-locus"><i class="fa fa-check"></i><b>2.2.6</b> More than one locus</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#non-random-mating"><i class="fa fa-check"></i><b>2.3</b> Non-random mating</a><ul>
<li class="chapter" data-level="2.3.1" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#assortative-mating"><i class="fa fa-check"></i><b>2.3.1</b> Assortative mating</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html"><i class="fa fa-check"></i><b>3</b> Changes of Gene Frequency</a><ul>
<li class="chapter" data-level="3.1" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#migration"><i class="fa fa-check"></i><b>3.1</b> Migration</a></li>
<li class="chapter" data-level="3.2" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#mutation"><i class="fa fa-check"></i><b>3.2</b> Mutation</a><ul>
<li class="chapter" data-level="3.2.1" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#non-recurrent-mutation"><i class="fa fa-check"></i><b>3.2.1</b> Non-recurrent mutation</a></li>
<li class="chapter" data-level="3.2.2" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#recurrent-mutation"><i class="fa fa-check"></i><b>3.2.2</b> Recurrent mutation</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#selection"><i class="fa fa-check"></i><b>3.3</b> Selection</a><ul>
<li class="chapter" data-level="3.3.1" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#change-of-gene-frequency-under-selection"><i class="fa fa-check"></i><b>3.3.1</b> Change of gene frequency under selection</a></li>
<li class="chapter" data-level="3.3.2" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#effectiveness-of-selection"><i class="fa fa-check"></i><b>3.3.2</b> Effectiveness of selection</a></li>
<li class="chapter" data-level="3.3.3" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#number-of-generations-required"><i class="fa fa-check"></i><b>3.3.3</b> Number of generations required</a></li>
<li class="chapter" data-level="3.3.4" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#average-fitness-and-load"><i class="fa fa-check"></i><b>3.3.4</b> Average fitness and load</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#equilibria"><i class="fa fa-check"></i><b>3.4</b> Equilibria</a><ul>
<li class="chapter" data-level="3.4.1" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#balance-between-mutation-and-selection"><i class="fa fa-check"></i><b>3.4.1</b> Balance between mutation and selection</a></li>
<li class="chapter" data-level="3.4.2" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#changes-of-equilibrium"><i class="fa fa-check"></i><b>3.4.2</b> Changes of equilibrium</a></li>
<li class="chapter" data-level="3.4.3" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#selection-favouring-heterozygotes"><i class="fa fa-check"></i><b>3.4.3</b> Selection favouring heterozygotes</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#polymorphism"><i class="fa fa-check"></i><b>3.5</b> Polymorphism</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><i class="fa fa-check"></i><b>4</b> Small populations: I. Changes of Gene Frequency Under Simplfied Conditions</a><ul>
<li class="chapter" data-level="4.1" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#the-idealized-population"><i class="fa fa-check"></i><b>4.1</b> The idealized population</a></li>
<li class="chapter" data-level="4.2" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#sampling"><i class="fa fa-check"></i><b>4.2</b> Sampling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#variance-of-gene-frequency"><i class="fa fa-check"></i><b>4.2.1</b> Variance of gene frequency</a></li>
<li class="chapter" data-level="4.2.2" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#fixation"><i class="fa fa-check"></i><b>4.2.2</b> Fixation</a></li>
<li class="chapter" data-level="4.2.3" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#genotype-frequencies"><i class="fa fa-check"></i><b>4.2.3</b> Genotype frequencies</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#inbreeding"><i class="fa fa-check"></i><b>4.3</b> Inbreeding</a><ul>
<li class="chapter" data-level="4.3.1" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#inbreeding-in-the-idealized-population"><i class="fa fa-check"></i><b>4.3.1</b> Inbreeding in the idealized population</a></li>
<li class="chapter" data-level="4.3.2" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#variance-of-gene-frequency-1"><i class="fa fa-check"></i><b>4.3.2</b> Variance of gene frequency</a></li>
<li class="chapter" data-level="4.3.3" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#genotype-frequencies-1"><i class="fa fa-check"></i><b>4.3.3</b> Genotype frequencies</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html"><i class="fa fa-check"></i><b>5</b> Small Populations: II. Less Simplified Conditions</a><ul>
<li class="chapter" data-level="5.1" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#effective-population-size"><i class="fa fa-check"></i><b>5.1</b> Effective population size</a><ul>
<li class="chapter" data-level="5.1.1" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#exclusion-of-closely-related-matings"><i class="fa fa-check"></i><b>5.1.1</b> exclusion of closely related matings</a></li>
<li class="chapter" data-level="5.1.2" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#different-numbers-of-males-and-females"><i class="fa fa-check"></i><b>5.1.2</b> Different numbers of males and females</a></li>
<li class="chapter" data-level="5.1.3" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#unequal-numbers-in-successive-generations"><i class="fa fa-check"></i><b>5.1.3</b> Unequal numbers in successive generations</a></li>
<li class="chapter" data-level="5.1.4" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#non-random-distribution-of-family-size"><i class="fa fa-check"></i><b>5.1.4</b> Non-random distribution of family size</a></li>
<li class="chapter" data-level="5.1.5" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#minimal-inbreeding"><i class="fa fa-check"></i><b>5.1.5</b> Minimal inbreeding</a></li>
<li class="chapter" data-level="5.1.6" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#overlapping-generations"><i class="fa fa-check"></i><b>5.1.6</b> Overlapping generations</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#mutation-migration-and-selection"><i class="fa fa-check"></i><b>5.2</b> Mutation, migration, and selection</a><ul>
<li class="chapter" data-level="5.2.1" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#non-recurrent-neutral-mutation"><i class="fa fa-check"></i><b>5.2.1</b> Non-recurrent neutral mutation</a></li>
<li class="chapter" data-level="5.2.2" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#recureent-mutation-and-migration"><i class="fa fa-check"></i><b>5.2.2</b> Recureent mutation and migration</a></li>
<li class="chapter" data-level="5.2.3" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#selection-1"><i class="fa fa-check"></i><b>5.2.3</b> Selection</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#random-drift-in-natural-populations"><i class="fa fa-check"></i><b>5.3</b> Random drift in natural populations</a></li>
<li class="chapter" data-level="5.4" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#polymorphism-1"><i class="fa fa-check"></i><b>5.4</b> Polymorphism</a><ul>
<li class="chapter" data-level="5.4.1" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#neutral-theory"><i class="fa fa-check"></i><b>5.4.1</b> Neutral theory</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><i class="fa fa-check"></i><b>6</b> Small Populations: III. Pedigreed Populations and Close Inbreeding</a><ul>
<li class="chapter" data-level="6.1" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#pedigreed-populations"><i class="fa fa-check"></i><b>6.1</b> Pedigreed populations</a><ul>
<li class="chapter" data-level="6.1.1" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#the-inbreeding-coefficient-of-an-individual"><i class="fa fa-check"></i><b>6.1.1</b> The inbreeding coefficient of an individual</a></li>
<li class="chapter" data-level="6.1.2" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#coancestry-or-kinship"><i class="fa fa-check"></i><b>6.1.2</b> Coancestry or kinship</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#regular-systems-of-inbreeding"><i class="fa fa-check"></i><b>6.2</b> Regular systems of inbreeding</a><ul>
<li class="chapter" data-level="6.2.1" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#close-inbreeding"><i class="fa fa-check"></i><b>6.2.1</b> Close inbreeding</a></li>
<li class="chapter" data-level="6.2.2" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#fixation-1"><i class="fa fa-check"></i><b>6.2.2</b> Fixation</a></li>
<li class="chapter" data-level="6.2.3" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#repeated-backcrosses"><i class="fa fa-check"></i><b>6.2.3</b> Repeated backcrosses</a></li>
<li class="chapter" data-level="6.2.4" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#crosses-and-subsequent-generations"><i class="fa fa-check"></i><b>6.2.4</b> Crosses and subsequent generations</a></li>
<li class="chapter" data-level="6.2.5" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#mixed-inbreeding-and-crossing"><i class="fa fa-check"></i><b>6.2.5</b> Mixed inbreeding and crossing</a></li>
<li class="chapter" data-level="6.2.6" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#change-of-base-strutured-population"><i class="fa fa-check"></i><b>6.2.6</b> Change of base: strutured population</a></li>
<li class="chapter" data-level="6.2.7" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#mutation-1"><i class="fa fa-check"></i><b>6.2.7</b> Mutation</a></li>
<li class="chapter" data-level="6.2.8" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#selection-favouring-heterozygotes-1"><i class="fa fa-check"></i><b>6.2.8</b> Selection favouring heterozygotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="continuous-variation.html"><a href="continuous-variation.html"><i class="fa fa-check"></i><b>7</b> Continuous Variation</a><ul>
<li class="chapter" data-level="7.1" data-path="continuous-variation.html"><a href="continuous-variation.html#metric-characters"><i class="fa fa-check"></i><b>7.1</b> Metric characters</a></li>
<li class="chapter" data-level="7.2" data-path="continuous-variation.html"><a href="continuous-variation.html#properties-of-metric-characters"><i class="fa fa-check"></i><b>7.2</b> Properties of metric characters</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="values-and-means.html"><a href="values-and-means.html"><i class="fa fa-check"></i><b>8</b> Values and Means</a><ul>
<li class="chapter" data-level="8.1" data-path="values-and-means.html"><a href="values-and-means.html#population-mean"><i class="fa fa-check"></i><b>8.1</b> Population mean</a></li>
<li class="chapter" data-level="8.2" data-path="values-and-means.html"><a href="values-and-means.html#average-effect"><i class="fa fa-check"></i><b>8.2</b> Average effect</a></li>
<li class="chapter" data-level="8.3" data-path="values-and-means.html"><a href="values-and-means.html#breeding-value"><i class="fa fa-check"></i><b>8.3</b> Breeding value</a></li>
<li class="chapter" data-level="8.4" data-path="values-and-means.html"><a href="values-and-means.html#dominance-deviation"><i class="fa fa-check"></i><b>8.4</b> Dominance deviation</a></li>
<li class="chapter" data-level="8.5" data-path="values-and-means.html"><a href="values-and-means.html#interaction-deviation"><i class="fa fa-check"></i><b>8.5</b> Interaction deviation</a></li>
<li class="chapter" data-level="8.6" data-path="values-and-means.html"><a href="values-and-means.html#note-proof-of-the-slope-of-regression-line"><i class="fa fa-check"></i><b>8.6</b> Note: Proof of the slope of regression line</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="variance.html"><a href="variance.html"><i class="fa fa-check"></i><b>9</b> Variance</a><ul>
<li class="chapter" data-level="9.1" data-path="variance.html"><a href="variance.html#components-of-variance"><i class="fa fa-check"></i><b>9.1</b> Components of variance</a><ul>
<li class="chapter" data-level="9.1.1" data-path="variance.html"><a href="variance.html#components-as-proportions-of-the-total"><i class="fa fa-check"></i><b>9.1.1</b> Components as proportions of the total</a></li>
<li class="chapter" data-level="9.1.2" data-path="variance.html"><a href="variance.html#estimation-of-the-degree-of-genetic-determination-v_gv_p"><i class="fa fa-check"></i><b>9.1.2</b> Estimation of the degree of genetic determination, <span class="math inline">\(V_G/V_P\)</span></a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="variance.html"><a href="variance.html#genetic-components-of-variance"><i class="fa fa-check"></i><b>9.2</b> Genetic components of variance</a><ul>
<li class="chapter" data-level="9.2.1" data-path="variance.html"><a href="variance.html#additive-and-dominance-variance"><i class="fa fa-check"></i><b>9.2.1</b> Additive and dominance variance</a></li>
<li class="chapter" data-level="9.2.2" data-path="variance.html"><a href="variance.html#total-genetic-variance"><i class="fa fa-check"></i><b>9.2.2</b> Total genetic variance</a></li>
<li class="chapter" data-level="9.2.3" data-path="variance.html"><a href="variance.html#interaction-variance"><i class="fa fa-check"></i><b>9.2.3</b> Interaction variance</a></li>
<li class="chapter" data-level="9.2.4" data-path="variance.html"><a href="variance.html#variance-due-to-disequilibrium"><i class="fa fa-check"></i><b>9.2.4</b> Variance due to disequilibrium</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="variance.html"><a href="variance.html#correlation-and-interaction-between-genotype-and-environment"><i class="fa fa-check"></i><b>9.3</b> Correlation and interaction between genotype and environment</a><ul>
<li class="chapter" data-level="9.3.1" data-path="variance.html"><a href="variance.html#correlation"><i class="fa fa-check"></i><b>9.3.1</b> Correlation</a></li>
<li class="chapter" data-level="9.3.2" data-path="variance.html"><a href="variance.html#interaction"><i class="fa fa-check"></i><b>9.3.2</b> Interaction</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="variance.html"><a href="variance.html#environmental-variance"><i class="fa fa-check"></i><b>9.4</b> Environmental variance</a><ul>
<li class="chapter" data-level="9.4.1" data-path="variance.html"><a href="variance.html#multiple-measurements-repeatability"><i class="fa fa-check"></i><b>9.4.1</b> Multiple measurements: repeatability</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="variance.html"><a href="variance.html#summary-of-variance-partitioning"><i class="fa fa-check"></i><b>9.5</b> Summary of variance partitioning</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html"><i class="fa fa-check"></i><b>10</b> Resemblance between relatives</a><ul>
<li class="chapter" data-level="10.1" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#genetic-covariance"><i class="fa fa-check"></i><b>10.1</b> Genetic covariance</a><ul>
<li class="chapter" data-level="10.1.1" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#offspring-and-one-parent"><i class="fa fa-check"></i><b>10.1.1</b> Offspring and one parent</a></li>
<li class="chapter" data-level="10.1.2" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#offspring-and-mid-parent"><i class="fa fa-check"></i><b>10.1.2</b> Offspring and mid-parent</a></li>
<li class="chapter" data-level="10.1.3" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#half-sibs"><i class="fa fa-check"></i><b>10.1.3</b> Half sibs</a></li>
<li class="chapter" data-level="10.1.4" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#full-sibs"><i class="fa fa-check"></i><b>10.1.4</b> Full sibs</a></li>
<li class="chapter" data-level="10.1.5" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#twins"><i class="fa fa-check"></i><b>10.1.5</b> Twins</a></li>
<li class="chapter" data-level="10.1.6" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#general"><i class="fa fa-check"></i><b>10.1.6</b> General</a></li>
<li class="chapter" data-level="10.1.7" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#epistatic-interaction"><i class="fa fa-check"></i><b>10.1.7</b> Epistatic interaction</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#environmental-covariance"><i class="fa fa-check"></i><b>10.2</b> Environmental covariance</a></li>
<li class="chapter" data-level="10.3" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#phenotypic-resemblance"><i class="fa fa-check"></i><b>10.3</b> Phenotypic resemblance</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="heritability.html"><a href="heritability.html"><i class="fa fa-check"></i><b>11</b> Heritability</a><ul>
<li class="chapter" data-level="11.1" data-path="heritability.html"><a href="heritability.html#estimation-of-heritability"><i class="fa fa-check"></i><b>11.1</b> Estimation of heritability</a><ul>
<li class="chapter" data-level="11.1.1" data-path="heritability.html"><a href="heritability.html#offspring-parent-regression"><i class="fa fa-check"></i><b>11.1.1</b> Offspring-parent regression</a></li>
<li class="chapter" data-level="11.1.2" data-path="heritability.html"><a href="heritability.html#sib-analysis"><i class="fa fa-check"></i><b>11.1.2</b> Sib analysis</a></li>
<li class="chapter" data-level="11.1.3" data-path="heritability.html"><a href="heritability.html#intra-sire-regression-of-offspring-on-dam"><i class="fa fa-check"></i><b>11.1.3</b> Intra-sire regression of offspring on dam</a></li>
<li class="chapter" data-level="11.1.4" data-path="heritability.html"><a href="heritability.html#combined-estimates"><i class="fa fa-check"></i><b>11.1.4</b> Combined estimates</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="heritability.html"><a href="heritability.html#twins-and-human-data"><i class="fa fa-check"></i><b>11.2</b> Twins and human data</a></li>
<li class="chapter" data-level="11.3" data-path="heritability.html"><a href="heritability.html#assortative-mating-1"><i class="fa fa-check"></i><b>11.3</b> Assortative mating</a></li>
<li class="chapter" data-level="11.4" data-path="heritability.html"><a href="heritability.html#precision-of-estimates-and-design-of-experiments"><i class="fa fa-check"></i><b>11.4</b> Precision of estimates and design of experiments</a><ul>
<li class="chapter" data-level="11.4.1" data-path="heritability.html"><a href="heritability.html#offspring-parent-regression-1"><i class="fa fa-check"></i><b>11.4.1</b> Offspring-parent regression</a></li>
<li class="chapter" data-level="11.4.2" data-path="heritability.html"><a href="heritability.html#sib-analysis-1"><i class="fa fa-check"></i><b>11.4.2</b> Sib analysis</a></li>
<li class="chapter" data-level="11.4.3" data-path="heritability.html"><a href="heritability.html#selection-of-parents"><i class="fa fa-check"></i><b>11.4.3</b> Selection of parents</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
Published with <a href="https://github.com/rstudio/bookdown" target="blank">bookdown</a> by <a href="http://zjuwhw.github.io/" target="blank">zjuwhw</a>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="genetic-constitution-of-a-population" class="section level1">
<h1><span class="header-section-number">2</span> Genetic Constitution of A Population</h1>
<div id="frequencies-of-genesalleles-and-genotypes" class="section level2">
<h2><span class="header-section-number">2.1</span> Frequencies of genes(alleles) and genotypes</h2>
<p><strong>Genotype frequencies</strong>: the freuqncy of a particular genotype being its proportion or percentage among the individuals.</p>
<p><strong>Population</strong>: in the genetic sense, a population is not just a group of individuals, but a breeding group; and the genetics of a population is concerned not only with the genetic constitution of the individuals but also with the transmission of the genes from one generation to the next.</p>
<p><strong>The genetic constitution of a population</strong>, refering to the genes it carries, is described by the array of <strong>gene(alleles) frequencies</strong>; that is, by specification of the alleles present at every locus and the numbers or proportions of the different alleles at each locus.</p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="center"></th>
<th align="center">Frequencies</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Genes(Alleles)</td>
<td align="center"><span class="math inline">\(A_1\)</span></td>
<td align="center">p</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_2\)</span></td>
<td align="center">q</td>
</tr>
<tr class="odd">
<td align="left">Genotypes</td>
<td align="center"><span class="math inline">\(A_1A_1\)</span></td>
<td align="center">P</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_1A_2\)</span></td>
<td align="center">H</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_2A_2\)</span></td>
<td align="center">Q</td>
</tr>
</tbody>
</table>
<ul>
<li>p + q =1</li>
<li>P + H + Q = 1</li>
<li><span class="math inline">\(p = P + \frac{1}{2}H\)</span> and <span class="math inline">\(q = Q+ \frac{1}{2}H\)</span></li>
</ul>
<div id="mendelian-variation-in-natural-populations" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Mendelian variation in natural populations</h3>
<ul>
<li>large effects vs small effects</li>
<li>restriction fragment length polymorphisms, or RFLPs</li>
<li>Microsatllite (or simple sequence repeat, SSR)</li>
<li>polymorphic loci</li>
</ul>
</div>
<div id="causes-of-change" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Causes of change</h3>
<p>The agencies through which the genetic properties of a population may be changed are these:</p>
<ul>
<li>Population size</li>
<li>Differences of fertility(繁殖能力) and viability(生存能力): selection</li>
<li>Migration and mutation</li>
<li>Mating system</li>
<li>random mating (or panmixia) means that any individual has an equal chance of mating with any other individual in the population.</li>
</ul>
</div>
</div>
<div id="hardy-weinberg-equilibrium" class="section level2">
<h2><span class="header-section-number">2.2</span> Hardy-Weinberg equilibrium</h2>
<div id="the-hardy-weinberg-law" class="section level3">
<h3><span class="header-section-number">2.2.1</span> The Hardy-Weinberg law</h3>
<p>In a large random-mating population with no selection, mutation, or migration, the gene frequencies and the genotype frequenies are constant from generation to generation; and, furthermore, there is a simple relationship between the gene frequencies and the genotype frequencies. These properties of a population are derived from a theorem, or principle, known as the <strong>Hardy-Weinberg law</strong> after Hardy and Weinberg, who independently demonstrated them in 1908. A population with constant gene and genotype frequencies is said to be in <strong>Hardy-Weinberg equilibrium</strong>.</p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="center"></th>
<th align="center">Frequencies</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Genes(Alleles)</td>
<td align="center"><span class="math inline">\(A_1\)</span></td>
<td align="center">p</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_2\)</span></td>
<td align="center">q</td>
</tr>
<tr class="odd">
<td align="left">Genotypes</td>
<td align="center"><span class="math inline">\(A_1A_1\)</span></td>
<td align="center"><span class="math inline">\(p^2\)</span></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_1A_2\)</span></td>
<td align="center"><span class="math inline">\(2pq\)</span></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_2A_2\)</span></td>
<td align="center"><span class="math inline">\(q^2\)</span></td>
</tr>
</tbody>
</table>
<p>Note: the relationship above refers to autosomal genes</p>
<p>Two additional conditions:</p>
<ul>
<li>genes segregate normally in gametogenesis</li>
<li>gene frequencies are the same in males and females</li>
</ul>
<p>Table 1.1 Steps of deduction in the proof of HWL, and the conditions that mst hold</p>
<table style="width:53%;">
<colgroup>
<col width="13%" />
<col width="12%" />
<col width="13%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Step</th>
<th align="right">Frequency</th>
<th align="center">deduction from:to</th>
<th align="center">Conditions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">0</td>
<td align="right">gene(allele)</td>
<td align="center">parents</td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="right">1a</td>
<td align="right">gene(allele)</td>
<td align="center">all gametes</td>
<td align="center">(1)Normal gene segregating</td>
</tr>
<tr class="odd">
<td align="right"></td>
<td align="right"></td>
<td align="center"></td>
<td align="center">(2)Equal fertility of parents</td>
</tr>
<tr class="even">
<td align="right">1b</td>
<td align="right">gene(allele)</td>
<td align="center">gametes forming zygotes</td>
<td align="center">(3)Equal fertilizing capacity of gametes</td>
</tr>
<tr class="odd">
<td align="right"></td>
<td align="right"></td>
<td align="center"></td>
<td align="center">(4)Large population</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="right">genotype</td>
<td align="center">zygotes</td>
<td align="center">(5)Random mating</td>
</tr>
<tr class="odd">
<td align="right"></td>
<td align="right"></td>
<td align="center"></td>
<td align="center">(6)Equal gene freuqncies in male and female parents</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="right">genotype</td>
<td align="center">progeny</td>
<td align="center">(7)Equal viability</td>
</tr>
<tr class="odd">
<td align="right">4</td>
<td align="right">gene(allele)</td>
<td align="center">progeny</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">p =<span class="st"> </span><span class="kw">seq</span>(<span class="dv">0</span>,<span class="dv">1</span>,<span class="fl">0.01</span>)
<span class="kw">plot</span>(p, (<span class="dv">1</span><span class="op">-</span>p)<span class="op">^</span><span class="dv">2</span>, <span class="dt">ylab =</span> <span class="st">&quot;genotype frequency&quot;</span>, <span class="dt">xlab=</span><span class="st">&quot;Gene Freuqncy of A2&quot;</span>, <span class="dt">main=</span><span class="st">&quot;Under HWE&quot;</span>,<span class="dt">type=</span><span class="st">&quot;l&quot;</span>, <span class="dt">col=</span><span class="st">&quot;blue&quot;</span>)
<span class="kw">lines</span>(p, <span class="dv">2</span><span class="op">*</span>p<span class="op">*</span>(<span class="dv">1</span><span class="op">-</span>p), <span class="dt">col=</span><span class="st">&quot;green&quot;</span>)
<span class="kw">lines</span>(p, p<span class="op">^</span><span class="dv">2</span>, <span class="dt">col=</span><span class="st">&quot;grey&quot;</span>)
<span class="kw">text</span>(<span class="fl">0.2</span>,<span class="fl">0.8</span>,<span class="st">&quot;A1A1&quot;</span>)
<span class="kw">text</span>(<span class="fl">0.5</span>,<span class="fl">0.45</span>,<span class="st">&quot;A1A2&quot;</span>)
<span class="kw">text</span>(<span class="fl">0.8</span>,<span class="fl">0.8</span>,<span class="st">&quot;A2A2&quot;</span>)</code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-1-1.png" width="672" /></p>
<p>Two important features:</p>
<ul>
<li>the maximum of heterozygote frequency is 50% when p = q = 0.5</li>
<li>the rare allele occures predominantly in heterozygotes</li>
</ul>
</div>
<div id="applications-of-the-hardy-weinberg-law" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Applications of the Hardy-Weinberg law</h3>
<ul>
<li>allele frequency</li>
<li>freuqncy of “carriers”</li>
</ul>
<p>For example, <span class="math inline">\(A_2A_2\)</span> is associated with a recessive phenotype, the <span class="math inline">\(A_2\)</span> allele frequency is q. So the normal heterozygotes frquency is 2q(1-q). The percent of heterozygotes among normal individuals is</p>
<p><span class="math display">\[H&#39; = \frac{2q(1-q)}{2q(1-q) + (1-q)^2} = \frac{2q}{1+q}\]</span> - Test of Hardy-Weinberg equilibrium</p>
</div>
<div id="mating-frequencies-and-another-proof-of-the-hardy-weinberg-law" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Mating frequencies and another proof of the Hardy-Weinberg law</h3>
<p>Table 1.3 Nine types of mating</p>
<table>
<thead>
<tr class="header">
<th align="center"></th>
<th align="center"><span class="math inline">\(A_1A_1\)</span>(P)</th>
<th align="center"><span class="math inline">\(A_1A_2\)</span>(H)</th>
<th align="center"><span class="math inline">\(A_2A_2\)</span>(Q)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><span class="math inline">\(A_1A_1\)</span>(P)</td>
<td align="center"><span class="math inline">\(P^2\)</span></td>
<td align="center">PH</td>
<td align="center">PQ</td>
</tr>
<tr class="even">
<td align="center"><span class="math inline">\(A_1A_2\)</span>(H)</td>
<td align="center">PH</td>
<td align="center"><span class="math inline">\(H^2\)</span></td>
<td align="center">HQ</td>
</tr>
<tr class="odd">
<td align="center"><span class="math inline">\(A_2A_2\)</span>(Q)</td>
<td align="center">PQ</td>
<td align="center">HQ</td>
<td align="center"><span class="math inline">\(Q^2\)</span></td>
</tr>
</tbody>
</table>
<p>Table 1.4</p>
<table style="width:35%;">
<colgroup>
<col width="6%" />
<col width="6%" />
<col width="6%" />
<col width="6%" />
<col width="6%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">mating type</th>
<th align="center">mating freuqncy</th>
<th align="center">frequency <span class="math inline">\(A_1A_1\)</span></th>
<th align="center">frequency <span class="math inline">\(A_1A_2\)</span></th>
<th align="center">frequency <span class="math inline">\(A_2A_2\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><span class="math inline">\(A_1A_1 \times A_1A_1\)</span></td>
<td align="center"><span class="math inline">\(P^2\)</span></td>
<td align="center"><span class="math inline">\(P^2\)</span></td>
<td align="center">-</td>
<td align="center">-</td>
</tr>
<tr class="even">
<td align="center"><span class="math inline">\(A_1A_1 \times A_1A_2\)</span></td>
<td align="center">2PH</td>
<td align="center">PH</td>
<td align="center">PH</td>
<td align="center">-</td>
</tr>
<tr class="odd">
<td align="center"><span class="math inline">\(A_1A_1 \times A_2A_2\)</span></td>
<td align="center">2PQ</td>
<td align="center">-</td>
<td align="center">2PQ</td>
<td align="center">-</td>
</tr>
<tr class="even">
<td align="center"><span class="math inline">\(A_1A_2 \times A_1A_2\)</span></td>
<td align="center"><span class="math inline">\(H^2\)</span></td>
<td align="center"><span class="math inline">\(\frac{1}{4}H^2\)</span></td>
<td align="center"><span class="math inline">\(\frac{1}{4}H^2\)</span></td>
<td align="center"><span class="math inline">\(\frac{1}{2}H^2\)</span></td>
</tr>
<tr class="odd">
<td align="center"><span class="math inline">\(A_1A_2 \times A_2A_2\)</span></td>
<td align="center">2HQ</td>
<td align="center">-</td>
<td align="center">HQ</td>
<td align="center">HQ</td>
</tr>
<tr class="even">
<td align="center"><span class="math inline">\(A_2A_2 \times A_2A_2\)</span></td>
<td align="center"><span class="math inline">\(Q^2\)</span></td>
<td align="center">-</td>
<td align="center">-</td>
<td align="center"><span class="math inline">\(Q^2\)</span></td>
</tr>
<tr class="odd">
<td align="center"></td>
<td align="center">Sums</td>
<td align="center"><span class="math inline">\(p^2\)</span></td>
<td align="center">2pq</td>
<td align="center"><span class="math inline">\(q^2\)</span></td>
</tr>
</tbody>
</table>
</div>
<div id="multiple-alleles" class="section level3">
<h3><span class="header-section-number">2.2.4</span> Multiple alleles</h3>
<p>When there are more than 2 alleles at a locus, the genotype frequencies are determined by the gene(allele) frequencies in exactly the same way as with two alleles.</p>
</div>
<div id="sex-linked-genes" class="section level3">
<h3><span class="header-section-number">2.2.5</span> Sex-linked genes</h3>
<p>For homogametic sex (human females), gene and genotype frequency is the same as with the autosomal gene, but the heterogametic sex (human males) has only two genotypes and each individual carries only one gene instead of two.</p>
<p>So 2/3 (4 out of 6) sex-linked alleles are carried by females and 1/3 by males in the population.</p>
<p>Consider two alleles, A1 and A2, with frequencies p and q, and the genotypic frequencies as follow:</p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="center"></th>
<th align="center">Frequencies</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Females</td>
<td align="center"><span class="math inline">\(A_1A_1\)</span></td>
<td align="center">P</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_1A_2\)</span></td>
<td align="center">H</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_2A_2\)</span></td>
<td align="center">Q</td>
</tr>
<tr class="even">
<td align="left">Males</td>
<td align="center"><span class="math inline">\(A_1\)</span></td>
<td align="center">R</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="center"><span class="math inline">\(A_2\)</span></td>
<td align="center">S</td>
</tr>
</tbody>
</table>
<p>For females, the A1 frequncy <span class="math inline">\(p_f = P + \frac{1}{2}H\)</span>; for males, the <span class="math inline">\(p_m = R\)</span>, so the frequency of <span class="math inline">\(A_1\)</span> in the whole population is,</p>
<p><span class="math display">\[\bar{p} = \frac{2}{3}p_f + \frac{1}{3}p_m = \frac{1}{3}(2P + H + R)\]</span> Males get their sex-linked genes only from their mothers, and females get their sex-linked genes equally from both parents. So</p>
<p><span class="math display">\[p_m^{&#39;} =p_f\text{ and }p_f^{&#39;}=\frac{1}{2}(p_m + p_f)\]</span></p>
<p>where, <span class="math inline">\(p_m^{&#39;}\)</span> and <span class="math inline">\(p_f^{&#39;}\)</span> are the gene frequnce in the progeny generation.</p>
<p>The differnce between the frequencies in the two sexes is</p>
<p><span class="math display">\[p_f^{&#39;}-p_m^{&#39;} = \frac{1}{2}(p_m + p_f) -p_f = -\frac{1}{2}(p_f - p_m)\]</span></p>
<p>Therefore the distribution of the genes between the two sexes oscillates, but the difference is halved in successive generations and the population rapidly approaches an equilibrium in which the frequencies in the two sexs are equal.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">GetGeneFreq_whole =<span class="st"> </span><span class="cf">function</span>(pf, pm){
  ### We assume the number of males and females are equal in the population
  <span class="kw">return</span>(<span class="dv">2</span><span class="op">/</span><span class="dv">3</span><span class="op">*</span>pf <span class="op">+</span><span class="st"> </span><span class="dv">1</span><span class="op">/</span><span class="dv">3</span><span class="op">*</span>pm)
}
GetGeneFreq_sex_next =<span class="st"> </span><span class="cf">function</span>(pf,pm){
  pm_next =<span class="st"> </span>pf
  pf_next =<span class="st"> </span><span class="dv">1</span><span class="op">/</span><span class="dv">2</span><span class="op">*</span>(pm<span class="op">+</span>pf)
  <span class="kw">return</span>(<span class="kw">c</span>(pf_next, pm_next))
}
data =<span class="st"> </span><span class="kw">c</span>()
pf =<span class="st"> </span><span class="dv">1</span>; pm =<span class="st"> </span><span class="dv">0</span>; gene=<span class="dv">0</span>
data =<span class="st"> </span><span class="kw">rbind</span>(data, <span class="kw">data.frame</span>(<span class="dt">gene=</span>gene, <span class="dt">female=</span>pf, <span class="dt">male=</span>pm, <span class="dt">whole=</span><span class="kw">GetGeneFreq_whole</span>(pf,pm)))
<span class="cf">for</span>(gene <span class="cf">in</span> <span class="dv">1</span><span class="op">:</span><span class="dv">10</span>){
  pfm =<span class="st"> </span><span class="kw">GetGeneFreq_sex_next</span>(pf,pm)
  pf =<span class="st"> </span>pfm[<span class="dv">1</span>]; pm=pfm[<span class="dv">2</span>]
  data =<span class="st"> </span><span class="kw">rbind</span>(data, <span class="kw">data.frame</span>(<span class="dt">gene=</span>gene, <span class="dt">female=</span>pf, <span class="dt">male=</span>pm, <span class="dt">whole=</span><span class="kw">GetGeneFreq_whole</span>(pf,pm)))
}
<span class="kw">library</span>(ggplot2)
<span class="kw">library</span>(reshape2)
data.melt =<span class="st"> </span><span class="kw">melt</span>(data, <span class="dt">id.vars =</span> <span class="st">&quot;gene&quot;</span>)
<span class="kw">ggplot</span>(data.melt, <span class="kw">aes</span>(<span class="dt">x=</span>gene, <span class="dt">y=</span>value,<span class="dt">colour=</span>variable), <span class="dt">group =</span> variable) <span class="op">+</span><span class="st"> </span><span class="kw">geom_line</span>() <span class="op">+</span>
<span class="st">  </span><span class="kw">ggtitle</span>(<span class="st">&quot;Approach to equilibrium under random mating for a sex-linked gene&quot;</span>) <span class="op">+</span><span class="st"> </span>
<span class="st">  </span><span class="kw">xlab</span>(<span class="st">&quot;Generation&quot;</span>) <span class="op">+</span>
<span class="st">  </span><span class="kw">ylab</span>(<span class="st">&quot;Gene(allele) Frequency&quot;</span>)</code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-2-1.png" width="672" /></p>
</div>
<div id="more-than-one-locus" class="section level3">
<h3><span class="header-section-number">2.2.6</span> More than one locus</h3>
<p>Disequilibrium with respect to two or more loci is called <strong>gametic phase disequilibrium</strong>, or <strong>linkage disequilibrium</strong>, irrespective of whether the loci are linked or not.</p>
<p>Disequilibrium can arise from intermixture of populations with different gene frequencies, or from chance in small populations. Disequilibrium can also be produced, and maintained, by selection favouring one combination of alleles over another.</p>
<p>Table 1.5</p>
<table>
<thead>
<tr class="header">
<th align="left">Genes(alleles)</th>
<th align="center"><span class="math inline">\(A_1\)</span></th>
<th align="center"><span class="math inline">\(A_2\)</span></th>
<th align="center"><span class="math inline">\(B_1\)</span></th>
<th align="center"><span class="math inline">\(B_2\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Gene/allele frequencies</td>
<td align="center"><span class="math inline">\(p_A\)</span></td>
<td align="center"><span class="math inline">\(q_A\)</span></td>
<td align="center"><span class="math inline">\(P_B\)</span></td>
<td align="center"><span class="math inline">\(q_B\)</span></td>
</tr>
<tr class="even">
<td align="left">Gametic types</td>
<td align="center"><span class="math inline">\(A_1B_1\)</span></td>
<td align="center"><span class="math inline">\(A_1B_2\)</span></td>
<td align="center"><span class="math inline">\(A_2B_1\)</span></td>
<td align="center"><span class="math inline">\(A_2B_2\)</span></td>
</tr>
<tr class="odd">
<td align="left">Frequencies, equilibrium</td>
<td align="center"><span class="math inline">\(p_Ap_B\)</span></td>
<td align="center"><span class="math inline">\(p_Aq_B\)</span></td>
<td align="center"><span class="math inline">\(q_Ap_B\)</span></td>
<td align="center"><span class="math inline">\(q_Aq_B\)</span></td>
</tr>
<tr class="even">
<td align="left">Frequencies, actual</td>
<td align="center">r</td>
<td align="center">s</td>
<td align="center">t</td>
<td align="center">u</td>
</tr>
<tr class="odd">
<td align="left">Difference from equilibrium</td>
<td align="center">+D</td>
<td align="center">-D</td>
<td align="center">-D</td>
<td align="center">+D</td>
</tr>
</tbody>
</table>
<p>Another way of Table 1.5</p>
<table>
<thead>
<tr class="header">
<th align="center"></th>
<th align="center"><span class="math inline">\(A_1(p_A)\)</span></th>
<th align="center"><span class="math inline">\(A_2(q_A)\)</span></th>
<th align="center">Total</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><span class="math inline">\(B_1(p_B)\)</span></td>
<td align="center"><span class="math inline">\(A_1B_1(r=p_Ap_B+D)\)</span></td>
<td align="center"><span class="math inline">\(A_2B_1(t=q_Ap_B-D)\)</span></td>
<td align="center"><span class="math inline">\(r+t=p_B\)</span></td>
</tr>
<tr class="even">
<td align="center"><span class="math inline">\(B_2(q_B)\)</span></td>
<td align="center"><span class="math inline">\(A_1B_2(s=p_Aq_B-D)\)</span></td>
<td align="center"><span class="math inline">\(A_2B_2(u=q_Aq_B+D)\)</span></td>
<td align="center"><span class="math inline">\(s+u=q_B\)</span></td>
</tr>
<tr class="odd">
<td align="center">Total</td>
<td align="center"><span class="math inline">\(r+s=p_A\)</span></td>
<td align="center"><span class="math inline">\(t+u=q_A\)</span></td>
<td align="center">1</td>
</tr>
</tbody>
</table>
<p><span class="math display">\[D = ru-st\]</span></p>
<p>Note: The statistic analysis “<span class="math inline">\(\chi^2_{[1]}=0.93; P&gt;0.5\)</span>” seems to me is wrong. Because <code>1-pchisq(0.93,1)=0.3348629</code>, it’s also not significance, but less than 0.5.</p>
<p>The maximum of the absolute value of D is 0.25 when <span class="math inline">\(p_A = q_A = p_B = q_B = 0.5\)</span>.</p>
<p><span class="math inline">\(D_{max}\)</span> is the largest numerical value D coud be, given the observed gene fequncies.</p>
<p>If two loci are linked on the same chromosome, the gamete type <span class="math inline">\(A_1B_1\)</span> in the progeny generation can be obtainded from <span class="math inline">\(A_1B_1/A_xB_x\)</span> without recombination or <span class="math inline">\(A_1B_x/A_xB_1\)</span> with recombination. The frequency of the former situation is r(1-c), r being the frequency of <span class="math inline">\(A_1B_1\)</span> in the parental gamets and c the recombination frequncy. The frequency of later situation is <span class="math inline">\(p_Ap_Bc\)</span>. therefore the frequency of <span class="math inline">\(A_1B_1\)</span> in the progeny gamets is</p>
<p><span class="math display">\[r&#39; = r(1-c) + p_Ap_Bc\]</span></p>
<p>and the disequilibrium in the progeny generation is</p>
<p><span class="math display">\[D&#39; = r&#39; - p_Ap_B = D(1-c)\]</span></p>
<p>In a more general way, after any number t of generations, the disequilibrium is given by</p>
<p><span class="math display">\[D_t = D_0 (1-c)^t\]</span></p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">plot</span>(<span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">12</span>),<span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="dt">col=</span><span class="st">&quot;white&quot;</span>,<span class="dt">main=</span><span class="st">&quot;Approach to equilibrium under random mating of two loci, considered jointly.&quot;</span>,<span class="dt">xlab=</span><span class="st">&quot;Generation&quot;</span>, <span class="dt">ylab=</span><span class="st">&quot;D&quot;</span>)
t =<span class="st"> </span><span class="dv">0</span><span class="op">:</span><span class="dv">12</span>;d0 =<span class="st"> </span><span class="dv">1</span>
<span class="kw">lines</span>(t,d0<span class="op">*</span>(<span class="dv">1</span><span class="op">-</span><span class="fl">0.05</span>)<span class="op">^</span>t, <span class="dt">col=</span><span class="dv">1</span>);<span class="kw">text</span>(<span class="dv">10</span>,<span class="fl">0.7</span>,<span class="st">&quot;0.05&quot;</span>)
<span class="kw">lines</span>(t,d0<span class="op">*</span>(<span class="dv">1</span><span class="op">-</span><span class="fl">0.1</span>)<span class="op">^</span>t, <span class="dt">col=</span><span class="dv">2</span>);<span class="kw">text</span>(<span class="dv">9</span>,<span class="fl">0.47</span>,<span class="st">&quot;0.1&quot;</span>)
<span class="kw">lines</span>(t,d0<span class="op">*</span>(<span class="dv">1</span><span class="op">-</span><span class="fl">0.2</span>)<span class="op">^</span>t, <span class="dt">col=</span><span class="dv">3</span>);<span class="kw">text</span>(<span class="dv">7</span>,<span class="fl">0.3</span>,<span class="st">&quot;0.2&quot;</span>)
<span class="kw">lines</span>(t,d0<span class="op">*</span>(<span class="dv">1</span><span class="op">-</span><span class="fl">0.3</span>)<span class="op">^</span>t, <span class="dt">col=</span><span class="dv">4</span>);<span class="kw">text</span>(<span class="dv">6</span>,<span class="fl">0.2</span>,<span class="st">&quot;0.3&quot;</span>)
<span class="kw">lines</span>(t,d0<span class="op">*</span>(<span class="dv">1</span><span class="op">-</span><span class="fl">0.5</span>)<span class="op">^</span>t, <span class="dt">col=</span><span class="dv">5</span>);<span class="kw">text</span>(<span class="fl">4.5</span>,<span class="fl">0.1</span>,<span class="st">&quot;0.5&quot;</span>)</code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-3-1.png" width="672" /></p>
</div>
</div>
<div id="non-random-mating" class="section level2">
<h2><span class="header-section-number">2.3</span> Non-random mating</h2>
<p>There are two distinct forms of non-random mating.</p>
<ul>
<li>The first is when mated individuals are related to each other by ancestral descent. This tends to increase the frequencies of homozygotes at all loci.</li>
<li>The second is when individuals tend to mate preferentially with respect to their genotypes at any particular locus under consideration.</li>
</ul>
<p>If mated pairs are of the same phenotype more often than would occur by chance, this is called <strong>assortative mating</strong>, and if less often, it is called <strong>disassortative mating</strong>.</p>
<div id="assortative-mating" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Assortative mating</h3>
<p>The effect of assortative mating on genotype frequencies among progeny is to increase the frequencies of homzygotes and reduce that of heterozygotes.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="changes-of-gene-frequency.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "none"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
