<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title></title>
  <meta name="description" content="">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="values-and-means.html">
<link rel="next" href="resemblance-between-relatives.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Notebook for QG</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Preface</a></li>
<li class="chapter" data-level="2" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html"><i class="fa fa-check"></i><b>2</b> Genetic Constitution of A Population</a><ul>
<li class="chapter" data-level="2.1" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#frequencies-of-genesalleles-and-genotypes"><i class="fa fa-check"></i><b>2.1</b> Frequencies of genes(alleles) and genotypes</a><ul>
<li class="chapter" data-level="2.1.1" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#mendelian-variation-in-natural-populations"><i class="fa fa-check"></i><b>2.1.1</b> Mendelian variation in natural populations</a></li>
<li class="chapter" data-level="2.1.2" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#causes-of-change"><i class="fa fa-check"></i><b>2.1.2</b> Causes of change</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#hardy-weinberg-equilibrium"><i class="fa fa-check"></i><b>2.2</b> Hardy-Weinberg equilibrium</a><ul>
<li class="chapter" data-level="2.2.1" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#the-hardy-weinberg-law"><i class="fa fa-check"></i><b>2.2.1</b> The Hardy-Weinberg law</a></li>
<li class="chapter" data-level="2.2.2" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#applications-of-the-hardy-weinberg-law"><i class="fa fa-check"></i><b>2.2.2</b> Applications of the Hardy-Weinberg law</a></li>
<li class="chapter" data-level="2.2.3" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#mating-frequencies-and-another-proof-of-the-hardy-weinberg-law"><i class="fa fa-check"></i><b>2.2.3</b> Mating frequencies and another proof of the Hardy-Weinberg law</a></li>
<li class="chapter" data-level="2.2.4" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#multiple-alleles"><i class="fa fa-check"></i><b>2.2.4</b> Multiple alleles</a></li>
<li class="chapter" data-level="2.2.5" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#sex-linked-genes"><i class="fa fa-check"></i><b>2.2.5</b> Sex-linked genes</a></li>
<li class="chapter" data-level="2.2.6" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#more-than-one-locus"><i class="fa fa-check"></i><b>2.2.6</b> More than one locus</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#non-random-mating"><i class="fa fa-check"></i><b>2.3</b> Non-random mating</a><ul>
<li class="chapter" data-level="2.3.1" data-path="genetic-constitution-of-a-population.html"><a href="genetic-constitution-of-a-population.html#assortative-mating"><i class="fa fa-check"></i><b>2.3.1</b> Assortative mating</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html"><i class="fa fa-check"></i><b>3</b> Changes of Gene Frequency</a><ul>
<li class="chapter" data-level="3.1" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#migration"><i class="fa fa-check"></i><b>3.1</b> Migration</a></li>
<li class="chapter" data-level="3.2" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#mutation"><i class="fa fa-check"></i><b>3.2</b> Mutation</a><ul>
<li class="chapter" data-level="3.2.1" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#non-recurrent-mutation"><i class="fa fa-check"></i><b>3.2.1</b> Non-recurrent mutation</a></li>
<li class="chapter" data-level="3.2.2" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#recurrent-mutation"><i class="fa fa-check"></i><b>3.2.2</b> Recurrent mutation</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#selection"><i class="fa fa-check"></i><b>3.3</b> Selection</a><ul>
<li class="chapter" data-level="3.3.1" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#change-of-gene-frequency-under-selection"><i class="fa fa-check"></i><b>3.3.1</b> Change of gene frequency under selection</a></li>
<li class="chapter" data-level="3.3.2" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#effectiveness-of-selection"><i class="fa fa-check"></i><b>3.3.2</b> Effectiveness of selection</a></li>
<li class="chapter" data-level="3.3.3" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#number-of-generations-required"><i class="fa fa-check"></i><b>3.3.3</b> Number of generations required</a></li>
<li class="chapter" data-level="3.3.4" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#average-fitness-and-load"><i class="fa fa-check"></i><b>3.3.4</b> Average fitness and load</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#equilibria"><i class="fa fa-check"></i><b>3.4</b> Equilibria</a><ul>
<li class="chapter" data-level="3.4.1" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#balance-between-mutation-and-selection"><i class="fa fa-check"></i><b>3.4.1</b> Balance between mutation and selection</a></li>
<li class="chapter" data-level="3.4.2" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#changes-of-equilibrium"><i class="fa fa-check"></i><b>3.4.2</b> Changes of equilibrium</a></li>
<li class="chapter" data-level="3.4.3" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#selection-favouring-heterozygotes"><i class="fa fa-check"></i><b>3.4.3</b> Selection favouring heterozygotes</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="changes-of-gene-frequency.html"><a href="changes-of-gene-frequency.html#polymorphism"><i class="fa fa-check"></i><b>3.5</b> Polymorphism</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><i class="fa fa-check"></i><b>4</b> Small populations: I. Changes of Gene Frequency Under Simplfied Conditions</a><ul>
<li class="chapter" data-level="4.1" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#the-idealized-population"><i class="fa fa-check"></i><b>4.1</b> The idealized population</a></li>
<li class="chapter" data-level="4.2" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#sampling"><i class="fa fa-check"></i><b>4.2</b> Sampling</a><ul>
<li class="chapter" data-level="4.2.1" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#variance-of-gene-frequency"><i class="fa fa-check"></i><b>4.2.1</b> Variance of gene frequency</a></li>
<li class="chapter" data-level="4.2.2" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#fixation"><i class="fa fa-check"></i><b>4.2.2</b> Fixation</a></li>
<li class="chapter" data-level="4.2.3" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#genotype-frequencies"><i class="fa fa-check"></i><b>4.2.3</b> Genotype frequencies</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#inbreeding"><i class="fa fa-check"></i><b>4.3</b> Inbreeding</a><ul>
<li class="chapter" data-level="4.3.1" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#inbreeding-in-the-idealized-population"><i class="fa fa-check"></i><b>4.3.1</b> Inbreeding in the idealized population</a></li>
<li class="chapter" data-level="4.3.2" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#variance-of-gene-frequency-1"><i class="fa fa-check"></i><b>4.3.2</b> Variance of gene frequency</a></li>
<li class="chapter" data-level="4.3.3" data-path="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html"><a href="small-populations-i-changes-of-gene-frequency-under-simplfied-conditions.html#genotype-frequencies-1"><i class="fa fa-check"></i><b>4.3.3</b> Genotype frequencies</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html"><i class="fa fa-check"></i><b>5</b> Small Populations: II. Less Simplified Conditions</a><ul>
<li class="chapter" data-level="5.1" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#effective-population-size"><i class="fa fa-check"></i><b>5.1</b> Effective population size</a><ul>
<li class="chapter" data-level="5.1.1" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#exclusion-of-closely-related-matings"><i class="fa fa-check"></i><b>5.1.1</b> exclusion of closely related matings</a></li>
<li class="chapter" data-level="5.1.2" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#different-numbers-of-males-and-females"><i class="fa fa-check"></i><b>5.1.2</b> Different numbers of males and females</a></li>
<li class="chapter" data-level="5.1.3" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#unequal-numbers-in-successive-generations"><i class="fa fa-check"></i><b>5.1.3</b> Unequal numbers in successive generations</a></li>
<li class="chapter" data-level="5.1.4" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#non-random-distribution-of-family-size"><i class="fa fa-check"></i><b>5.1.4</b> Non-random distribution of family size</a></li>
<li class="chapter" data-level="5.1.5" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#minimal-inbreeding"><i class="fa fa-check"></i><b>5.1.5</b> Minimal inbreeding</a></li>
<li class="chapter" data-level="5.1.6" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#overlapping-generations"><i class="fa fa-check"></i><b>5.1.6</b> Overlapping generations</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#mutation-migration-and-selection"><i class="fa fa-check"></i><b>5.2</b> Mutation, migration, and selection</a><ul>
<li class="chapter" data-level="5.2.1" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#non-recurrent-neutral-mutation"><i class="fa fa-check"></i><b>5.2.1</b> Non-recurrent neutral mutation</a></li>
<li class="chapter" data-level="5.2.2" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#recureent-mutation-and-migration"><i class="fa fa-check"></i><b>5.2.2</b> Recureent mutation and migration</a></li>
<li class="chapter" data-level="5.2.3" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#selection-1"><i class="fa fa-check"></i><b>5.2.3</b> Selection</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#random-drift-in-natural-populations"><i class="fa fa-check"></i><b>5.3</b> Random drift in natural populations</a></li>
<li class="chapter" data-level="5.4" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#polymorphism-1"><i class="fa fa-check"></i><b>5.4</b> Polymorphism</a><ul>
<li class="chapter" data-level="5.4.1" data-path="small-populations-ii-less-simplified-conditions.html"><a href="small-populations-ii-less-simplified-conditions.html#neutral-theory"><i class="fa fa-check"></i><b>5.4.1</b> Neutral theory</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><i class="fa fa-check"></i><b>6</b> Small Populations: III. Pedigreed Populations and Close Inbreeding</a><ul>
<li class="chapter" data-level="6.1" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#pedigreed-populations"><i class="fa fa-check"></i><b>6.1</b> Pedigreed populations</a><ul>
<li class="chapter" data-level="6.1.1" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#the-inbreeding-coefficient-of-an-individual"><i class="fa fa-check"></i><b>6.1.1</b> The inbreeding coefficient of an individual</a></li>
<li class="chapter" data-level="6.1.2" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#coancestry-or-kinship"><i class="fa fa-check"></i><b>6.1.2</b> Coancestry or kinship</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#regular-systems-of-inbreeding"><i class="fa fa-check"></i><b>6.2</b> Regular systems of inbreeding</a><ul>
<li class="chapter" data-level="6.2.1" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#close-inbreeding"><i class="fa fa-check"></i><b>6.2.1</b> Close inbreeding</a></li>
<li class="chapter" data-level="6.2.2" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#fixation-1"><i class="fa fa-check"></i><b>6.2.2</b> Fixation</a></li>
<li class="chapter" data-level="6.2.3" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#repeated-backcrosses"><i class="fa fa-check"></i><b>6.2.3</b> Repeated backcrosses</a></li>
<li class="chapter" data-level="6.2.4" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#crosses-and-subsequent-generations"><i class="fa fa-check"></i><b>6.2.4</b> Crosses and subsequent generations</a></li>
<li class="chapter" data-level="6.2.5" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#mixed-inbreeding-and-crossing"><i class="fa fa-check"></i><b>6.2.5</b> Mixed inbreeding and crossing</a></li>
<li class="chapter" data-level="6.2.6" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#change-of-base-strutured-population"><i class="fa fa-check"></i><b>6.2.6</b> Change of base: strutured population</a></li>
<li class="chapter" data-level="6.2.7" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#mutation-1"><i class="fa fa-check"></i><b>6.2.7</b> Mutation</a></li>
<li class="chapter" data-level="6.2.8" data-path="small-populations-iii-pedigreed-populations-and-close-inbreeding.html"><a href="small-populations-iii-pedigreed-populations-and-close-inbreeding.html#selection-favouring-heterozygotes-1"><i class="fa fa-check"></i><b>6.2.8</b> Selection favouring heterozygotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="continuous-variation.html"><a href="continuous-variation.html"><i class="fa fa-check"></i><b>7</b> Continuous Variation</a><ul>
<li class="chapter" data-level="7.1" data-path="continuous-variation.html"><a href="continuous-variation.html#metric-characters"><i class="fa fa-check"></i><b>7.1</b> Metric characters</a></li>
<li class="chapter" data-level="7.2" data-path="continuous-variation.html"><a href="continuous-variation.html#properties-of-metric-characters"><i class="fa fa-check"></i><b>7.2</b> Properties of metric characters</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="values-and-means.html"><a href="values-and-means.html"><i class="fa fa-check"></i><b>8</b> Values and Means</a><ul>
<li class="chapter" data-level="8.1" data-path="values-and-means.html"><a href="values-and-means.html#population-mean"><i class="fa fa-check"></i><b>8.1</b> Population mean</a></li>
<li class="chapter" data-level="8.2" data-path="values-and-means.html"><a href="values-and-means.html#average-effect"><i class="fa fa-check"></i><b>8.2</b> Average effect</a></li>
<li class="chapter" data-level="8.3" data-path="values-and-means.html"><a href="values-and-means.html#breeding-value"><i class="fa fa-check"></i><b>8.3</b> Breeding value</a></li>
<li class="chapter" data-level="8.4" data-path="values-and-means.html"><a href="values-and-means.html#dominance-deviation"><i class="fa fa-check"></i><b>8.4</b> Dominance deviation</a></li>
<li class="chapter" data-level="8.5" data-path="values-and-means.html"><a href="values-and-means.html#interaction-deviation"><i class="fa fa-check"></i><b>8.5</b> Interaction deviation</a></li>
<li class="chapter" data-level="8.6" data-path="values-and-means.html"><a href="values-and-means.html#note-proof-of-the-slope-of-regression-line"><i class="fa fa-check"></i><b>8.6</b> Note: Proof of the slope of regression line</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="variance.html"><a href="variance.html"><i class="fa fa-check"></i><b>9</b> Variance</a><ul>
<li class="chapter" data-level="9.1" data-path="variance.html"><a href="variance.html#components-of-variance"><i class="fa fa-check"></i><b>9.1</b> Components of variance</a><ul>
<li class="chapter" data-level="9.1.1" data-path="variance.html"><a href="variance.html#components-as-proportions-of-the-total"><i class="fa fa-check"></i><b>9.1.1</b> Components as proportions of the total</a></li>
<li class="chapter" data-level="9.1.2" data-path="variance.html"><a href="variance.html#estimation-of-the-degree-of-genetic-determination-v_gv_p"><i class="fa fa-check"></i><b>9.1.2</b> Estimation of the degree of genetic determination, <span class="math inline">\(V_G/V_P\)</span></a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="variance.html"><a href="variance.html#genetic-components-of-variance"><i class="fa fa-check"></i><b>9.2</b> Genetic components of variance</a><ul>
<li class="chapter" data-level="9.2.1" data-path="variance.html"><a href="variance.html#additive-and-dominance-variance"><i class="fa fa-check"></i><b>9.2.1</b> Additive and dominance variance</a></li>
<li class="chapter" data-level="9.2.2" data-path="variance.html"><a href="variance.html#total-genetic-variance"><i class="fa fa-check"></i><b>9.2.2</b> Total genetic variance</a></li>
<li class="chapter" data-level="9.2.3" data-path="variance.html"><a href="variance.html#interaction-variance"><i class="fa fa-check"></i><b>9.2.3</b> Interaction variance</a></li>
<li class="chapter" data-level="9.2.4" data-path="variance.html"><a href="variance.html#variance-due-to-disequilibrium"><i class="fa fa-check"></i><b>9.2.4</b> Variance due to disequilibrium</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="variance.html"><a href="variance.html#correlation-and-interaction-between-genotype-and-environment"><i class="fa fa-check"></i><b>9.3</b> Correlation and interaction between genotype and environment</a><ul>
<li class="chapter" data-level="9.3.1" data-path="variance.html"><a href="variance.html#correlation"><i class="fa fa-check"></i><b>9.3.1</b> Correlation</a></li>
<li class="chapter" data-level="9.3.2" data-path="variance.html"><a href="variance.html#interaction"><i class="fa fa-check"></i><b>9.3.2</b> Interaction</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="variance.html"><a href="variance.html#environmental-variance"><i class="fa fa-check"></i><b>9.4</b> Environmental variance</a><ul>
<li class="chapter" data-level="9.4.1" data-path="variance.html"><a href="variance.html#multiple-measurements-repeatability"><i class="fa fa-check"></i><b>9.4.1</b> Multiple measurements: repeatability</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="variance.html"><a href="variance.html#summary-of-variance-partitioning"><i class="fa fa-check"></i><b>9.5</b> Summary of variance partitioning</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html"><i class="fa fa-check"></i><b>10</b> Resemblance between relatives</a><ul>
<li class="chapter" data-level="10.1" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#genetic-covariance"><i class="fa fa-check"></i><b>10.1</b> Genetic covariance</a><ul>
<li class="chapter" data-level="10.1.1" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#offspring-and-one-parent"><i class="fa fa-check"></i><b>10.1.1</b> Offspring and one parent</a></li>
<li class="chapter" data-level="10.1.2" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#offspring-and-mid-parent"><i class="fa fa-check"></i><b>10.1.2</b> Offspring and mid-parent</a></li>
<li class="chapter" data-level="10.1.3" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#half-sibs"><i class="fa fa-check"></i><b>10.1.3</b> Half sibs</a></li>
<li class="chapter" data-level="10.1.4" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#full-sibs"><i class="fa fa-check"></i><b>10.1.4</b> Full sibs</a></li>
<li class="chapter" data-level="10.1.5" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#twins"><i class="fa fa-check"></i><b>10.1.5</b> Twins</a></li>
<li class="chapter" data-level="10.1.6" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#general"><i class="fa fa-check"></i><b>10.1.6</b> General</a></li>
<li class="chapter" data-level="10.1.7" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#epistatic-interaction"><i class="fa fa-check"></i><b>10.1.7</b> Epistatic interaction</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#environmental-covariance"><i class="fa fa-check"></i><b>10.2</b> Environmental covariance</a></li>
<li class="chapter" data-level="10.3" data-path="resemblance-between-relatives.html"><a href="resemblance-between-relatives.html#phenotypic-resemblance"><i class="fa fa-check"></i><b>10.3</b> Phenotypic resemblance</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="heritability.html"><a href="heritability.html"><i class="fa fa-check"></i><b>11</b> Heritability</a><ul>
<li class="chapter" data-level="11.1" data-path="heritability.html"><a href="heritability.html#estimation-of-heritability"><i class="fa fa-check"></i><b>11.1</b> Estimation of heritability</a><ul>
<li class="chapter" data-level="11.1.1" data-path="heritability.html"><a href="heritability.html#offspring-parent-regression"><i class="fa fa-check"></i><b>11.1.1</b> Offspring-parent regression</a></li>
<li class="chapter" data-level="11.1.2" data-path="heritability.html"><a href="heritability.html#sib-analysis"><i class="fa fa-check"></i><b>11.1.2</b> Sib analysis</a></li>
<li class="chapter" data-level="11.1.3" data-path="heritability.html"><a href="heritability.html#intra-sire-regression-of-offspring-on-dam"><i class="fa fa-check"></i><b>11.1.3</b> Intra-sire regression of offspring on dam</a></li>
<li class="chapter" data-level="11.1.4" data-path="heritability.html"><a href="heritability.html#combined-estimates"><i class="fa fa-check"></i><b>11.1.4</b> Combined estimates</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="heritability.html"><a href="heritability.html#twins-and-human-data"><i class="fa fa-check"></i><b>11.2</b> Twins and human data</a></li>
<li class="chapter" data-level="11.3" data-path="heritability.html"><a href="heritability.html#assortative-mating-1"><i class="fa fa-check"></i><b>11.3</b> Assortative mating</a></li>
<li class="chapter" data-level="11.4" data-path="heritability.html"><a href="heritability.html#precision-of-estimates-and-design-of-experiments"><i class="fa fa-check"></i><b>11.4</b> Precision of estimates and design of experiments</a><ul>
<li class="chapter" data-level="11.4.1" data-path="heritability.html"><a href="heritability.html#offspring-parent-regression-1"><i class="fa fa-check"></i><b>11.4.1</b> Offspring-parent regression</a></li>
<li class="chapter" data-level="11.4.2" data-path="heritability.html"><a href="heritability.html#sib-analysis-1"><i class="fa fa-check"></i><b>11.4.2</b> Sib analysis</a></li>
<li class="chapter" data-level="11.4.3" data-path="heritability.html"><a href="heritability.html#selection-of-parents"><i class="fa fa-check"></i><b>11.4.3</b> Selection of parents</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
Published with <a href="https://github.com/rstudio/bookdown" target="blank">bookdown</a> by <a href="http://zjuwhw.github.io/" target="blank">zjuwhw</a>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="variance" class="section level1">
<h1><span class="header-section-number">9</span> Variance</h1>
<p>The basic idea in the study of variation is its partitioning into components attributable to different causes.</p>
<div id="components-of-variance" class="section level2">
<h2><span class="header-section-number">9.1</span> Components of variance</h2>
<p>The total variance is then, with certain qualifications (条件), the sum of the components, thus:</p>
<p><span class="math display">\[V_P = V_G+V_E = V_A + V_D + V_I + V_E\]</span></p>
<p>The qualifications:</p>
<ul>
<li>Cov(G,E) = 0. If it’s not zero, it will increase <span class="math inline">\(V_P\)</span> by <span class="math inline">\(2\times Cov(G,E)\)</span></li>
<li>there is no interaction between genotypes and environments.</li>
</ul>
<p>Table 8.1 Components of variance</p>
<table>
<thead>
<tr class="header">
<th align="left">Variance component</th>
<th align="left">Symbol</th>
<th align="left">Value whose variance is measured</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Phenotypic</td>
<td align="left"><span class="math inline">\(V_P\)</span></td>
<td align="left">Phenotypic value</td>
</tr>
<tr class="even">
<td align="left">Genotypic</td>
<td align="left"><span class="math inline">\(V_G\)</span></td>
<td align="left">Genotypic value</td>
</tr>
<tr class="odd">
<td align="left">Additive</td>
<td align="left"><span class="math inline">\(V_A\)</span></td>
<td align="left">Breeding value</td>
</tr>
<tr class="even">
<td align="left">Domiance</td>
<td align="left"><span class="math inline">\(V_D\)</span></td>
<td align="left">Dominance deviation</td>
</tr>
<tr class="odd">
<td align="left">Interaction</td>
<td align="left"><span class="math inline">\(V_I\)</span></td>
<td align="left">Interaction deviation</td>
</tr>
<tr class="even">
<td align="left">Environmental</td>
<td align="left"><span class="math inline">\(V_E\)</span></td>
<td align="left">Environmental deviation</td>
</tr>
</tbody>
</table>
<div id="components-as-proportions-of-the-total" class="section level3">
<h3><span class="header-section-number">9.1.1</span> Components as proportions of the total</h3>
<p>The partitioning of the variance into its components allows us to estimate the relative importance of the various determinants of the phenotype, in particular the role of heredity versus environment, or nature and nurture.</p>
<p>The relative importance of heredity in determining phenotypic values is called the <strong>heritability</strong> of the character.</p>
<p>The ratio <span class="math inline">\(V_G/V_P\)</span> expresses the extent to which individuals’ phenotypes are determined by the genotypes. This is called the <strong>heritability in the broad sense</strong>, or <strong>the degree of genetic determination</strong>. The ratio <span class="math inline">\(V_A/V_P\)</span> expresses the extent to which phenotypes are determined by the genes transmitted from the parents. This is called the <strong>heritability in the narrow sense</strong>, or simply the <strong>heritability</strong>.</p>
<p>The heritability <span class="math inline">\(V_A/V_P\)</span> determines the degree of resemblance between relatives and is therefore of the greatest importance in breeding programmes. The degree of genetic determination <span class="math inline">\(V_G/V_P\)</span> is of more theoretical interest than practical importance.</p>
</div>
<div id="estimation-of-the-degree-of-genetic-determination-v_gv_p" class="section level3">
<h3><span class="header-section-number">9.1.2</span> Estimation of the degree of genetic determination, <span class="math inline">\(V_G/V_P\)</span></h3>
<p>Neither the genotypic nor the environmental components of variance, <span class="math inline">\(V_G\)</span> and <span class="math inline">\(V_E\)</span>, can be estimated directly from observations on a single populations, but in certain circumstances they can be estimated in experimental populations.</p>
<p>Individuals with identical genotypes can be obtained from a highly inbred line or the <span class="math inline">\(F_1\)</span> of a cross between two such lines, or from a clone propagated from a single individual. (Identical twins in man and cattle also provide individuals of identical genotype, but their use in partitioning the variance is very limited for reasons to be discussed in Chapter 10.)</p>
</div>
</div>
<div id="genetic-components-of-variance" class="section level2">
<h2><span class="header-section-number">9.2</span> Genetic components of variance</h2>
<div id="additive-and-dominance-variance" class="section level3">
<h3><span class="header-section-number">9.2.1</span> Additive and dominance variance</h3>
<p>The information needed to obtain expressions for the variance of breeding values and the variance of dominance deviations was given in the last chapter in Table 7.3. (Note: the expected of breeding values and domiance deviations are both 0.)</p>
<p>The additive variance, which is the vraince of breeding values, is obtained as follows:</p>
<p><span class="math display">\[V_A = p^2 \times (2q\alpha)^2 + 2pq \times ((q-p)\alpha)^2 + q^2 \times (-2p\alpha)^2 = 2pq[a+d(q-p)]^2\]</span></p>
<p>Similarly, the variance of domiance deviations is</p>
<p><span class="math display">\[V_D = p^2 \times (-2q^2d)^2 + 2pq \times (2pqd)^2 + q^2 \times (-2p^2d)^2 = (2pqd)^2\]</span></p>
</div>
<div id="total-genetic-variance" class="section level3">
<h3><span class="header-section-number">9.2.2</span> Total genetic variance</h3>
<p>So, the genotypic variance is</p>
<p><span class="math display">\[V_G = V_A + V_D = 2pq[a+d(q-p)]^2 + [2pqd]^2\]</span></p>
<p>Note: cov(A,D) = 0 due to the least square estimation.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">va =<span class="st"> </span><span class="cf">function</span>(a, d, p){
  q =<span class="st"> </span><span class="dv">1</span> <span class="op">-</span><span class="st"> </span>p
  <span class="kw">return</span>(<span class="dv">2</span><span class="op">*</span>p<span class="op">*</span>q<span class="op">*</span>(a<span class="op">+</span>d<span class="op">*</span>(q<span class="op">-</span>p))<span class="op">^</span><span class="dv">2</span>)
}
vd =<span class="st"> </span><span class="cf">function</span>(a, d, p){
  q =<span class="st"> </span><span class="dv">1</span> <span class="op">-</span><span class="st"> </span>p
  <span class="kw">return</span>((<span class="dv">2</span><span class="op">*</span>p <span class="op">*</span><span class="st"> </span>q <span class="op">*</span>d)<span class="op">^</span><span class="dv">2</span>)
}
vg =<span class="st"> </span><span class="cf">function</span>(a, d, p){
  <span class="kw">return</span>(<span class="kw">va</span>(a,d,p) <span class="op">+</span><span class="st"> </span><span class="kw">vd</span> (a,d,p))
}
p =<span class="st"> </span><span class="kw">seq</span>(<span class="dv">0</span>,<span class="dv">1</span>,<span class="fl">0.01</span>); q=<span class="dv">1</span><span class="op">-</span>p
<span class="kw">layout</span>(<span class="kw">matrix</span>(<span class="dv">1</span><span class="op">:</span><span class="dv">3</span>,<span class="dv">1</span>,<span class="dv">3</span>))

a=<span class="dv">1</span>; d=<span class="dv">0</span>
<span class="kw">plot</span>(<span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="dt">col =</span> <span class="st">&quot;white&quot;</span>, <span class="dt">main=</span><span class="kw">paste</span>(<span class="st">&quot;a=&quot;</span>,a,<span class="st">&quot;, d=&quot;</span>,d,<span class="dt">sep=</span><span class="st">&quot;&quot;</span>), <span class="dt">xlab=</span><span class="st">&quot;Gene(Allele) frequency, q&quot;</span>, <span class="dt">ylab=</span><span class="st">&quot;genetic components of variance&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">vg</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;red&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">vd</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;green&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">va</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;blue&quot;</span>)

a=<span class="dv">1</span>; d=<span class="dv">1</span>
<span class="kw">plot</span>(<span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="dt">col =</span> <span class="st">&quot;white&quot;</span>, <span class="dt">main=</span><span class="kw">paste</span>(<span class="st">&quot;a=&quot;</span>,a,<span class="st">&quot;, d=&quot;</span>,d,<span class="dt">sep=</span><span class="st">&quot;&quot;</span>), <span class="dt">xlab=</span><span class="st">&quot;Gene(Allele) frequency, q&quot;</span>, <span class="dt">ylab=</span><span class="st">&quot;genetic components of variance&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">vg</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;red&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">vd</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;green&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">va</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;blue&quot;</span>)

a=<span class="dv">0</span>; d=<span class="dv">1</span>
<span class="kw">plot</span>(<span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),<span class="dt">col =</span> <span class="st">&quot;white&quot;</span>, <span class="dt">main=</span><span class="kw">paste</span>(<span class="st">&quot;a=&quot;</span>,a,<span class="st">&quot;, d=&quot;</span>,d,<span class="dt">sep=</span><span class="st">&quot;&quot;</span>), <span class="dt">xlab=</span><span class="st">&quot;Gene(Allele) frequency, q&quot;</span>, <span class="dt">ylab=</span><span class="st">&quot;genetic components of variance&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">vg</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;red&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">vd</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;green&quot;</span>)
<span class="kw">lines</span>(q, <span class="kw">va</span>(a,d,p),<span class="dt">col=</span><span class="st">&quot;blue&quot;</span>)

<span class="kw">legend</span>(<span class="fl">0.1</span>,<span class="dv">1</span>,<span class="st">&quot;Genotypic variance&quot;</span>,<span class="dt">lty =</span> <span class="dv">1</span>, <span class="dt">col =</span><span class="st">&quot;red&quot;</span>, <span class="dt">bty=</span><span class="st">&quot;n&quot;</span>)
<span class="kw">legend</span>(<span class="fl">0.1</span>,<span class="fl">0.95</span>,<span class="st">&quot;Additive variance&quot;</span>,<span class="dt">lty =</span> <span class="dv">1</span>, <span class="dt">col =</span><span class="st">&quot;blue&quot;</span>, <span class="dt">bty=</span><span class="st">&quot;n&quot;</span>)
<span class="kw">legend</span>(<span class="fl">0.1</span>,<span class="fl">0.9</span>,<span class="st">&quot;Dominance variance&quot;</span>,<span class="dt">lty =</span> <span class="dv">1</span>, <span class="dt">col =</span><span class="st">&quot;green&quot;</span>, <span class="dt">bty=</span><span class="st">&quot;n&quot;</span>)</code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-5-1.png" width="672" /></p>
<p>The graphs show the amounts of genotypic, additive, domiance variance arising from a single locus with two alleles, plotted against the gene(allele) frequency in threes cases: 1) no dominance (d=0); 2) complete dominance (d=a); 3) “pure” overdominance.</p>
<p>The general conclusion to be drawn from these graphs is that genes contribute much more variance when at intermediate frequencies than when at high or low frequencies: recessives at low frequency, in particular, contribute very little variance.</p>
</div>
<div id="interaction-variance" class="section level3">
<h3><span class="header-section-number">9.2.3</span> Interaction variance</h3>
<p>If the genotypes at different loci show epistatic interaction, then the interactions give rise to a component of variance <span class="math inline">\(V_I\)</span>, which is the variance of the interaction deviations.</p>
<p>Theoretical description of the properties of interaction variances rests on its further subdivision into components.</p>
<p>It is first subdivided according to the number of loci involved: two-factor interaction arises from the interaction of two loci, three-factor from three loci, etc. Interactions involving larger numbers of loci contribute so little variance that they can be ignored.</p>
<p>The nxt subdivision of the interaction variance is according to whether the interaction involves breeding values or domiance deviations. There are thus three sorts of two-factor interactions. Interaction between the two breeding values gives rise to additive <span class="math inline">\(\times\)</span> additive variance, <span class="math inline">\(V_{AA}\)</span>; interaction between the breeding value of one locus and the domiance deviation of the other gives rise to additive <span class="math inline">\(\times\)</span> domiance variance, <span class="math inline">\(V_{AD}\)</span>; and interaction between the two domiance deviations give rise to domiance <span class="math inline">\(\times\)</span> domiance variance, <span class="math inline">\(V_{DD}\)</span>.</p>
<p>So,</p>
<p><span class="math display">\[V_I = V_{AA} + V_{AD} + V_{DD} + etc.\]</span></p>
<p>It is not easy, however, to estimate the amout of variance that it generates, and little is known about its relative importance as a source of variation. Th e experimental evidence is reviewd by Barker (1979). For further details of epistatic interaction, see Cockerham (1954, 1963), Kempthorne (1957), Crow and Kimura (1970).</p>
</div>
<div id="variance-due-to-disequilibrium" class="section level3">
<h3><span class="header-section-number">9.2.4</span> Variance due to disequilibrium</h3>
<p>The disequilibrium when the genotype freuqncies at two or more loci consideered jointly introduces an additional source of genetic variance. For simplicity, consider just tow loci which do not interact in the manner described above. Let G’ and G’‘be genotypic values of individuals with respect to each locus separately, and let G be the genotypic value with respect to both jointly, i.e., G = G’ + G’’. The total genotypic variance caused by the two loci together is then</p>
<p><span class="math display">\[V_G = V_{G&#39;} + V_{G&#39;&#39;} + 2 cov(G&#39;,G&#39;&#39;)\]</span></p>
</div>
</div>
<div id="correlation-and-interaction-between-genotype-and-environment" class="section level2">
<h2><span class="header-section-number">9.3</span> Correlation and interaction between genotype and environment</h2>
<div id="correlation" class="section level3">
<h3><span class="header-section-number">9.3.1</span> Correlation</h3>
<p><span class="math display">\[V_P = V_G + V_E + 2cov(G,E)\]</span></p>
</div>
<div id="interaction" class="section level3">
<h3><span class="header-section-number">9.3.2</span> Interaction</h3>
<p><span class="math display">\[P = G + E + I_{GE}\]</span></p>
<p><span class="math display">\[V_P = V_G + V_E + 2cov_{GE} + V_{GE}\]</span></p>
<p>The variance due to interaction is included with the environmental variance estimated from the phenotypic variance of that genotype.</p>
<p>To some extent, the environmental variance is a property of the genotype. But the source of the variation is environmental and not genetic. It is therefore logical, as well as experimentally necessary, to regard any variance due to genotype-environment interaction as being part of the environmental variance included in any estimate of <span class="math inline">\(V_E\)</span>.</p>
<div id="environmental-sensitivity" class="section level4">
<h4><span class="header-section-number">9.3.2.1</span> Environmental sensitivity</h4>
<p>Some of the genotype-environment interaction can be ascribed to differences of sensitivity of different genotypes. Then environmental sensitivity of a genotype is also known as its “<strong>reaction norm</strong>”.</p>
<p>The only way in which environments can be quantified is by the mean performance of all the genotypes. In other words, the measure of an environment is the mean of all genotypes in that environment. This will be called the <strong>environmental value</strong>.</p>
<p>The variance due to interaction of genotypes with the specific environments is estimated from an analysis of variance, and the amount attributable to differences of sensitivity is obtained from the heterogeneity of regeression slops.</p>
</div>
</div>
</div>
<div id="environmental-variance" class="section level2">
<h2><span class="header-section-number">9.4</span> Environmental variance</h2>
<p>Environmental variance, which by definition embraces all variation of non-genetic origin, can have a great variety of causes and its nature depends very much on the character and the organism studied.</p>
<ul>
<li>Nutritional（营养因素） and climatic（气候因素） factors</li>
<li>Maternal effects（母亲因素）</li>
<li>prenatal influences（出生前影响）</li>
<li>postnatal influences（出生后影响）</li>
<li>Error of measurement</li>
</ul>
<div id="multiple-measurements-repeatability" class="section level3">
<h3><span class="header-section-number">9.4.1</span> Multiple measurements: repeatability</h3>
<p>When more than one measurement of the character can be made on each individual, the phenotypic variance can be partitioned into variance within individuals and variance between individuals. This partitioning leads to a ratio of variance components called the <strong>repeatability</strong>.</p>
<ul>
<li>temporal（时间的） repetition</li>
<li>spatial（空间的） repetition</li>
</ul>
<p>In order that we may discuss both temporal and spatial repetition together, we shall use the terms <strong>special environmental variance</strong>, <span class="math inline">\(V_{Es}\)</span>, to refer to the within-individual variance arising from temporary or localized circumstances; and <strong>general environmental variance</strong>, <span class="math inline">\(V_{Eg}\)</span>, to refer to the environmental variance contributing to the between-individual component and arising from permanent or non-localized circumstances. The ratio of the between-individual component to the total phenotypic variance is the intraclass correlation r. It is the correlation between repeated measurements of the same individual, and is known as the repeatability of the character.</p>
<p><span class="math display">\[V_P = (V_G + V_{Eg}) + V_{Es}\]</span></p>
<p><span class="math display">\[r = \frac{V_G + V_{Eg}}{V_P}\]</span></p>
<p><span class="math display">\[\frac{V_{Es}}{V_P} = 1-r\]</span></p>
<p>It can be seen that the repeatability sets an upper limit to the degree of genetic determination <span class="math inline">\(V_G/V_P\)</span>, and to the heritability <span class="math inline">\(V_A/V_P\)</span>.</p>
</div>
</div>
<div id="summary-of-variance-partitioning" class="section level2">
<h2><span class="header-section-number">9.5</span> Summary of variance partitioning</h2>
<p>Table 8.5 Summary of variance partitioning</p>
<table style="width:17%;">
<colgroup>
<col width="5%" />
<col width="5%" />
<col width="5%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Data needed</th>
<th align="left">Partition made</th>
<th align="left">Ratio estimated</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Resemblance between relatives</td>
<td align="left"><span class="math inline">\((V_A):(V_{NA} + V_{Eg} + V_{Es})\)</span></td>
<td align="left">heritability, <span class="math inline">\(V_A/V_P\)</span></td>
</tr>
<tr class="even">
<td align="left">Genetically uniform group</td>
<td align="left"><span class="math inline">\((V_A + V_{NA}):(V_{Eg} + V_{Es}) = (V_G):(V_E)\)</span></td>
<td align="left">degree of genetic determination, <span class="math inline">\(V_G/V_P\)</span></td>
</tr>
<tr class="odd">
<td align="left">Multiple measurements</td>
<td align="left"><span class="math inline">\((V_G + V_{Eg}):V_{Es}\)</span></td>
<td align="left">repeatability (<span class="math inline">\(V_G+V_{Eg}/V_P\)</span>)</td>
</tr>
<tr class="even">
<td align="left">All three</td>
<td align="left"><span class="math inline">\(V_A:V_{NA}:V_{Eg}:V_{Es}\)</span></td>
<td align="left"></td>
</tr>
</tbody>
</table>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="values-and-means.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="resemblance-between-relatives.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "none"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
